intermediate_source/dynamic_quantization_bert_tutorial.rst

```diff
@@ -525,6 +528,21 @@ We can serialize and save the quantized model for the future use using
 
 .. code:: python
 
+    def ids_tensor(shape, vocab_size, rng=None, name=None):
+        #  Creates a random int32 tensor of the shape within the vocab size
+        if rng is None:
+            rng = global_rng
+
+        total_dims = 1
+        for dim in shape:
+            total_dims *= dim
+
+        values = []
+        for _ in range(total_dims):
+            values.append(rng.randint(0, vocab_size - 1))
+
+        return torch.tensor(data=values, dtype=torch.long, device='cpu').view(shape).contiguous()
```

Comment:
Also, isn't it all just equivalent of `return torch.randint(0, vocab_size, shape=shape, dtype=torch.int, device='cpu')

Reply:
https://github.com/pytorch/tutorials/pull/2431